app/javascript/packs/mirador.js

```diff
@@ -52,7 +54,15 @@ Mirador.viewer({
   workspaceControlPanel: {
     enabled: false,
   },
-  windows: [
-    {loadedManifest: manifestUrl}
-  ]
-})
+  windows: [{
+    imageToolsEnabled: true,
+    imageToolsOpen: true,
```

Comment:
@ggeisler : should we default to showing the whole toolbar or just the toggle? I'm not sure what the intended use for these tools is..

Reply:
![Screen Shot 2022-02-25 at 12 00 31](https://user-images.githubusercontent.com/1328900/155786992-26aaea69-2598-43be-b9f4-cc91361ac9b8.png)

Current view for reference with "open" as default

Reply:
Vs. closed default
![Screen Shot 2022-02-25 at 12 01 20](https://user-images.githubusercontent.com/1328900/155787185-e9b8afbf-1d4d-4f55-9690-78ad4ada024a.png)
t

Reply:
I suggest closed by default. I think the goal for DLME is to enable users who might have a use case for one or more of the tools to have access to them, but there are likely far more DLME users who don't have image-tool-related use cases. So the more unobtrusive closed version seems appropriate as the default.